Makefile

```diff
@@ -52,11 +52,11 @@ pylint:
 ###########
 
 snapshot:
-	skipper run python3 tools/update_assisted_installer_yaml.py --full
+	skipper run -i python3 tools/update_assisted_installer_yaml.py --full
```

Comment:
Why interactive?

Reply:
Sorry, didn't intend to push that
Thank you!